Content.Server/Store/Systems/StoreSystem.Ui.cs

```diff
@@ -200,8 +200,8 @@ private void OnRequestWithdraw(EntityUid uid, StoreComponent component, StoreReq
         {
             var cashId = proto.Cash[value];
             var amountToSpawn = (int) MathF.Floor((float) (amountRemaining / value));
-            var ent = _stack.SpawnMultiple(cashId, amountToSpawn, coordinates);
-            _hands.PickupOrDrop(buyer, ent);
+            var ents = _stack.SpawnMultiple(cashId, amountToSpawn, coordinates);
+            _hands.PickupOrDrop(buyer, ents.Last());
```

Comment:
Is `ents.Last()` a full stack or the partial stack? Whichever it is, the pickup should be a full stack if there is one.

Reply:
it was partial, but i modified it to be full.